### PR TITLE
fix(theme): align disabled tabs background color more with patternfly (5 for now)

### PR DESCRIPTION
**Pull request by christoph-jerolimov**
## Hey, I just made a Pull Request!

Always before and with this PR:

MUI v4 Light theme

image

image

MUI v4 Dark theme

image

image

MUI v5 Light theme

image

image

MUI v4 Dark theme

image

image

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/redhat-developer/rhdh-plugins/blob/main/CONTRIBUTING.md#creating-changesets))
- [ ] Added or Updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)

### CODE REVIEW

File: workspaces/theme/plugins/theme/src/utils/createComponents.ts
Comment (ciiay) [inline workspaces/theme/plugins/theme/src/utils/createComponents.ts:1]: Hi @christoph-jerolimov , I found an issue non-related to your fix but it looks bad on disabled tab missing the bottom border. We can update `boxShadow` on L424 with `borderBottom` to fix this issue, what do you think?

```
      styleOverrides: {
        root: {
          borderBottom: `1px solid ${general.tabsBottomBorderColor}`,
          padding: '0 1.5rem',
        },
```
image

**Review by ciiay — COMMENTED** [review]
It looks nice. Just found another small issue probably left by me when I worked on theme and I made a comment for the fix. Also since this PR is open, can you remove [L161](https://github.com/redhat-developer/rhdh-plugins/blob/main/workspaces/theme/plugins/theme/src/utils/createComponents.ts#L161) in `createComponents.ts`, just to clean the code as this line is a leftover and it doesn't do anything. Thank you 🙏

File: workspaces/theme/plugins/theme/src/utils/createComponents.ts
Comment (christoph-jerolimov) [reply workspaces/theme/plugins/theme/src/utils/createComponents.ts:1]: Applied that change. Thanks.

**Review by ciiay — APPROVED** [review]
/lgtm

Thank you 💯 Looks great!

**Pull request merged**